README.md

```diff
@@ -290,6 +290,23 @@ Use the `subscriptionCapability` subcommand to delete a capability from a subscr
 * Delete a capability from an organization `ocm support delete organizationCapability [orgID] [capabilityKey]`
 * Delete a capability from a subscription `ocm support delete subscriptionCapability [subscriptionID] [capabilityKey]`
 
+##### Deleting type independent capabilities
+
+Use the `capability` subcommand to delete a capability by passing the ID, or provide filter value to search matching capabilities and delete them. By default the dry run flag will be enabled. Pass `no-dry-run` flag to actually remove the resource.
+
+The following flags are available for `delete capability`:
+
+```
+--filter                     If passed, filters and deletes the matching capabilities.
+--no-dry-run                 If passed, it will execute the delete command call in instead of a dry run.
```

Comment:
I am wondering wheather **no-dry-run** is the best word choice for the flag. A **dry-run** flag set to true by default sounds more "natural" for the user and in the code. 

Thoughts on that?

Reply:
That sounds more logical. Changed the flag to dryRun